test/polyrith.lean

```diff
@@ -5,6 +5,8 @@ Authors: Dhruv Bhatia, Robert Y. Lewis, Mario Carneiro
 -/
 import Mathlib.Tactic.Polyrith
 
+set_option linter.longLine false
```

Comment:
Why this one?

Reply:
Most of this file is commented out and it is probably expected to be uncommented once more work on `polyrith` makes it valid again.  Silencing the linter once, instead of on each "offending" segment seemed less invasive.

Reply:
Optionally, you can write a comment (so future contributors modifying this file will know). Not something I will insist on, though.